tools/composer-require-checker/composer.json

```diff
@@ -3,7 +3,7 @@
     "type": "metapackage",
     "description": "composer-require-checker",
     "require-dev": {
-        "maglnet/composer-require-checker": "4.3.0",
+        "maglnet/composer-require-checker": "4.4.0",
```

Comment:
needed as v4.3 has issues with ext-rand beeing part of php's core. 
see https://github.com/maglnet/ComposerRequireChecker/issues/364 
read https://www.php.net/manual/en/random.installation.php